gsi_wdl_tools/workflow_info.py

```diff
@@ -67,9 +68,29 @@ def outputs(self):
                     description = output_descriptions.get(output_file_name)
             else:
                 raise Exception('Unsupported input type')
+            vidarr_label = ""
+            if wdl_type == "Pair[File,Map[String,String]]":
+                # Extracting existing entries from output.info.expr.right
+                existing_entries = output.info.expr.right.items
+
+                # Iterate through the items
+                for key, value in existing_entries:
+                    key_eval = key.eval(None, None)  # Provide env and stdlib as None
+                    value_eval = value.eval(None, None)
+                    
+                    # Check if the evaluated key matches "vidarr_label" string
+                    if key_eval.value.strip('"') == "vidarr_label":
```

Comment:
I don't know much about how this tool is used in practice, but if someone has manually written out a `Pair[File,Map[String,String]]`, I don't think they'd use 'vidarr_label' as a key name. In this case, might want to add all the values in that Map as Labels?